.flake8

```diff
@@ -1,6 +1,6 @@
 [flake8]
 max-line-length = 88
-extend-ignore = E203, E501
+extend-ignore = E203, E501, W291
```

Comment:
is this necessary?:)

Reply:
This might not be necessary anymore, I think it was needed when I converted jupyter to py files.

Reply:
Took it out!